drivers/DeepSmart/deepsmart/src/lifecycles.lua

```diff
@@ -0,0 +1,123 @@
+local config = require('config')
+local log = require('log')
+local caps = require('st.capabilities')
+local wisers = require('deepsmart.wisers')
+local lifecycle_handler = {}
+
+function lifecycle_handler.init(driver, device)
+  -- report test data
+  --wisers.default_report(driver, device)
+  local is_bridge = wisers.is_device_bridge(device)
+  log.info('device '..device.id..' in init')
+  -- for bridge do not reload devices&&config
+  -- just use the devices&&config loaded from wiser's datastore
+  -- only discover or refresh bridge will reload devices&&config
+  if (is_bridge) then
+    -- just add wiser(bridge) to wisers
+    wisers.add_wiser(device.device_network_id, device:get_field(config.FIELD.IP), device)
+    -- load devices
+    wisers.refresh_wiser(device.device_network_id)
+    -- refresh all devices
```

Comment:
add devices init in bridge lifecycle-init.

Reply:
> bridge is the last one to enter the function(as the order device1,device2...bridge)

This ordering is not a guarantee by the platform. I dont think that assumption is built in, but your comment suggests otherwise. Will everything still work if the bridge init event is handled before the device init events?

Reply:
Delete the init device and initialize the logical device when added